README.md

```diff
@@ -1 +1,69 @@
-# PictureViewMaster
\ No newline at end of file
+## PictureViewMaster
+Interactive image projector.
+
+
+[![Build Status](https://api.travis-ci.org/inaka/EventSource.svg)](https://travis-ci.org/inaka/EventSource)
+
+```
+This library supports Swift 2.3
+Swift 2.3 is currently in branch master.
+```
+
+### Abstract
+
+This is a library that allows UIImageViews (By subclassing them on the PictureMasterImageView class) to handle a touch that will enlarge the image and let the user play with them in the ways you enable. (Dragging, rotating, zooming)
+
+### How to use it?
+
+Instead of using regulars UIImageView objects use PictureMasterImageView. They'll automatically add a gesture recognizer that will let its delegate knows when the user tap on it. Then, on the delegate callback, implement the PictureMasterViewController by instantiating an object and giving it the UIImage and/or the gestures you want to support (All of them by default.). It will bring a view inside your current view with the large image and all the gesture recognizers, plus the dismissing functionality.
+NOTE: PictureMasterImageView is not actually required. You can just pass the UIImage you want to the `PictureMasterViewController` and it will also works. The UIImage subclass is just to have a nice and easy way to have clickable and enlargable UIImageViews. 
+
+### Install
+
+Drag this files to your project :
+`PictureMasterViewController.xib`
+`PictureMasterViewController.swift`
+`PictureMasterImageView.swift`
+and you are ready to go. 
+Or you can use CocoaPods:
+
+```
+pod 'PictureMasterView'
+
+```
+
+Then import the library:
+
+```
+import PictureMasterView
+```
+
+#### USAGE:
+
+```swift
+//Asign the delegate to the PictureMasterImageView object
+let sampleImage : PictureMasterImageView = PictureMasterImageView(image: UIImage(named:"sampleImage"))
+sampleImage.delegate = self
+
+//Then conform the the MasterViewDelegate
+func showImageInMasterView(image: UIImage) {
```

Comment:
Remember to update the readme once you've changed the code with the PR comments.